src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -151,7 +151,7 @@ def default_api_version(self):
         ResourceType.MGMT_COMPUTE: SDKProfile('2022-03-01', {
             'resource_skus': '2019-04-01',
             'disks': '2021-12-01',
-            'disk_encryption_sets': '2020-12-01',
+            'disk_encryption_sets': '2021-04-01',
```

Comment:
```suggestion
            'disk_encryption_sets': '2022-03-02',
```
Please keep consistent with the api-version for `disk_encryption_sets` #22780 used by other PR in this sprint